pom.xml

```diff
@@ -51,6 +51,7 @@
         <testcontainers.version>1.19.8</testcontainers.version>
         <javax.transaction-api.version>1.3</javax.transaction-api.version>
         <byte-buddy.version>1.14.17</byte-buddy.version>
+        <re2j.version>1.7</re2j.version>
```

Comment:
could you please move it to  pdfbox and okapi.xliff dependencies and comment these deps as <!-- Test dependencies --> ?

Reply:
Done. Please check whether it is what you've expected.